user_guide_src/source/cli/cli_library.rst

```diff
@@ -203,7 +203,7 @@ Would create something like this:
                text of the printing and
                typesetting industry.
     task1abc   Lorem Ipsum has been the
-               industry's standard dummy
+               industry standard dummy
```

Comment:
The apostrophe here is correct.

Reply:
ChatGPT said "industry standard" is correct.

The following is an answer by Claude 3 Sonnet.
Is this explanation correct?

---
Both "industry's standard" and "industry standard" are grammatically correct in English, but they have slightly different meanings and usages.

1. "Industry standard" (without an apostrophe)
This term refers to a standard, practice, or specification that is widely accepted and used within a particular industry. It is the norm or generally accepted way of doing something in that industry. For example:

- USB is an industry standard for connecting peripheral devices to computers.
- PDF has become an industry standard for sharing documents across different platforms.

2. "Industry's standard" (with an apostrophe)
This phrase refers to the specific standard or norm that is used or accepted by a particular industry. It is more explicit in indicating that the standard belongs to or is associated with a specific industry. For example:

- The construction industry's standard for safety equipment has become more stringent.
- The automotive industry's standard for fuel efficiency has been revised.

In most cases, "industry standard" without the apostrophe is more common and idiomatic when referring to a widely accepted norm or practice within an industry. However, "industry's standard" can be used when you want to emphasize that the standard is specific to a particular industry or clarify which industry you are referring to.

Reply:
From https://www.lipsum.com/

> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s [...]

In this case we can rewrite the 2nd sentence as "Lorem ipsum has been the standard dummy text of the [printing and typesetting] industry ever since the 1500s [...]"

Another way to look at it is this: industry standard is a noun; if we mean it that way, we cannot use it as an adjective for "dummy text"

Reply:
@paulbalandan Thank you. I got it.

I fixed the question, but ChatGPT still says "industry standard" is correct. It seems s/he like the phrase.
https://chatgpt.com/share/0c469d88-dc36-40ed-9ab9-b8c179db89b0
Gemini says "industry's standard" is grammatically correct.